icmd/command.go

```diff
@@ -252,6 +254,12 @@ func buildCmd(cmd Cmd) *Result {
 		execCmd.Stdout = outBuffer
 	}
 	execCmd.Stderr = errBuffer
+
+	if len(cmd.ExtraFiles) > 0 {
+		extraFiles := make([]*os.File, len(cmd.ExtraFiles))
+		copy(extraFiles, cmd.ExtraFiles)
```

Comment:
Is it necessary to copy the slice? I don't see anything in the Godoc that suggests it will be modified

Reply:
Yeah it's probably fine to not copy.

Reply:
Updated